flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/internal/JdbcBatchingOutputFormat.java

```diff
@@ -179,11 +178,9 @@ public synchronized void flush() throws IOException {
 					throw new IOException(e);
 				}
 				try {
-					if (!connection.isValid(CONNECTION_CHECK_TIMEOUT_SECONDS)) {
```

Comment:
I think we should keep this. Putting field `connectionCheckTimeoutSeconds ` in the `jdbcOptions` object maybe some unreasonable, for `jdbcOptions` holds information about database connections. I think it is better to put it in `JdbcLookupOptions` and `JdbcExecutionOptions`.

Reply:
WDYT @wuchong. If yes, could you cherry pick this commit @xiaoHoly https://github.com/apache/flink/commit/4f7d5aa207a121f6ca02569f5095fa5d52b8addb

Reply:
I would like to avoid to put this configuration in multiple places (we may also need this in reader in the future), otherwise, it is easy to have inconsistent configuration. 

Another idea is puting this in `JdbcConnectionOptions` as it belongs to connection scope.

Reply:
+1 to unify the scope and put it to `JdbcConnectionOptions `.
1. As it belongs to connection scope, so I think maybe we  should add a method `isConnectionValid` to `JdbcConnectionProvider` interface, and for this , `JdbcBatchingOutputFormat` will properly use this without adding this config to `executionOptions`.
2. For the lookupFunction and reader, we can get it directly from `JdbcConnectionOptions `.

Reply:
I think we should set this configuration in only one place ,this is the reason I remove the code in JdbcBatchingOutputFormat.java.
@wangxlong thanks for your work with your commit  wangxlong@4f7d5aa,according to your code i will migrate this configuration to JdbcExecutionOptions  .

Reply:
Let's put it in `JdbcConnectionOptions` and exposing `isConnectionValid` sounds good to me.

Reply:
yes,i find put it in JdbcExecutionOptions is not good ，It will cause jdbcOptions to fail to configuration .
+1 to put this configuration in JdbcConnectionOptions